### PR TITLE
ci: emit names of tests run in Travis

### PR DESCRIPTION
**Pull request by OddBloke**
This makes it easier to debug differences in test behaviour between
Travis and local developer environments.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,9 @@ matrix:
     fast_finish: true
     include:
         - python: 3.6
-          env: TOXENV=py3
+          env:
+              TOXENV=py3
+              NOSE_VERBOSE=2  # List all tests run by nose
```
Comment (igalic) [inline .travis.yml]: couldn't you set this env one level higher instead of explicitly in each test?

Comment (OddBloke) [reply .travis.yml]: I tried that initially, but it appears that the per-job `env` overrides the top-level `env`.

Comment (igalic) [reply .travis.yml]: i don't even know what to say to that

Comment (OddBloke) [reply .travis.yml]: "Approve"? :joy:

**Comment by blackboxsw** [comment]
Thanks Meena for review here. and +1 for me too. It's nice to see verbosity in all cases; success or failure.

**Pull request merged**